src/tsickle_compiler_host.ts

```diff
@@ -145,7 +145,7 @@ export class TsickleCompilerHost implements ts.CompilerHost {
       if (this.options.googmodule && !fileName.match(/\.d\.ts$/)) {
         content = this.convertCommonJsToGoogModule(fileName, content);
       }
-      if (this.tscOptions.inlineSourceMap) {
+      if (!/.*\.d\.ts/.test(path.basename(fileName)) && this.tscOptions.inlineSourceMap) {
```

Comment:
There's this `isDtsFileName` function in tsickle.ts, maybe we could reuse that?  (I noticed that the regex is unanchored here)